boule/_ellipsoid.py

```diff
@@ -182,6 +182,24 @@ def semiminor_axis(self):
         """
         return self.semimajor_axis * (1 - self.flattening)
 
+    @property
+    def semimedium_axis(self):
+        """
```

Comment:
```suggestion
        r"""
```
or?

Reply:
I don't think that we need a raw string here, as there aren't any backslashes in the latex commands.

Reply:
ah yes you are right